mapstory/settings/base.py

```diff
@@ -291,7 +291,7 @@ def str_to_bool(v):
     'osgeo_importer.handlers.geoserver.GeoServerTimeHandler',
     'osgeo_importer.handlers.geoserver.GeoWebCacheHandler',
     'osgeo_importer.handlers.geonode.GeoNodePublishHandler',
-    'osgeo_importer.handlers.geoserver.GenericSLDHandler',
+    'mapstory.import_handlers.DefaultStyleHandler',
```

Comment:
I think this SLD handler needs to happen before the GeonodePublish handler (or thumbnails will be created before the style is applied)

Reply:
order switched